### PR TITLE
[hplsql] adding 'HPL/SQL with 'Hive' in editor heading

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- adding HPL/SQL with Hive in editor heading

### CODE REVIEW

**Comment by agl29** [comment]
Screenshot 2021-10-01 at 6 42 32 PM

**Comment by agl29** [comment]
Any other better solution?

**Comment by Harshg999** [comment]
Nice change!
Also, maybe `Hive - PL/SQL` or `Hive (PL/SQL)`? Because I think H means Hive in HPL/SQL right?

**Comment by agl29** [comment]
> Nice change! Also, maybe `Hive - PL/SQL` or `Hive (PL/SQL)`? Because I think H means Hive in HPL/SQL right?

Nope, it is  Hive Hybrid Procedural SQL.

**Comment by Harshg999** [comment]
Ohh I see! Thanks!

File: desktop/core/src/desktop/js/apps/editor/EditorViewModel.js [hunk 1]
```diff
@@ -91,6 +91,9 @@ export default class EditorViewModel {
     this.editorTitle = ko.pureComputed(
       () => this.activeConnector() && this.activeConnector().displayName
     );
+    this._editorTitle = ko.pureComputed(() =>
```
Comment (JohanAhlen) [inline desktop/core/src/desktop/js/apps/editor/EditorViewModel.js:94]: Why the duplicate?

Also note that you'd have to add the same logic for editor v1.

Comment (agl29) [reply desktop/core/src/desktop/js/apps/editor/EditorViewModel.js]: Done.

**Review by JohanAhlen — APPROVED** [review]
Nice!

**Pull request merged**